#! /bin/sh /usr/share/dpatch/dpatch-run
## sb200506.dpatch by Richard James Laager (Richie Laager) <rlaager@wiktel.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Treasury Redemption Data File: sb200506

@DPATCH@

diff -urN gbonds-2.0.2-orig/data/sb200506.asc gbonds-2.0.2/data/sb200506.asc
--- gbonds-2.0.2-orig/data/sb200506.asc	1969-12-31 18:00:00.000000000 -0600
+++ gbonds-2.0.2/data/sb200506.asc	2006-04-21 20:34:22.000000000 -0500
@@ -0,0 +1,468 @@
+E2005061980009075009075009075009075009167009167                                    
+E2005061979009242009242009242009242009331009353009170009170009170009170009257009257
+E2005061978011406011406009729009729009472009496009312009312009312009312009400009426
+E2005061977012753012753012753012753012879012906012651012651012651012651011602011634
+E2005061976012953012953012953012953013076013105012846012846012846012846012971013008
+E2005061975013157013157013157013157013283013315013054013054013054013054013182013211
+E2005061974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005061973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005061972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005061971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005061970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005061969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005061968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005061967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005061966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005061965023411021474021579021577021577018597018324018042018135017840017840012812
+E2005061964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005061963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005061962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005061961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005061960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005061959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005061958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005061957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005061956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005061955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005061954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005061953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005061952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005061951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005061950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005061949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005061948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005061947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005061946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005061945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005061944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005061943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005061942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005061941                        009059009196009196009196009196009196009286009435
+S2005061970013671013671013671013671013804013870013870013870013870013870            
+S2005061969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005061968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005061967                        015333015404015405015405015405015405015552015628
+E2005071980009257009075009075009075009167009167                                    
+E2005071979009427009242009242009242009331009353009353009170009170009170009257009257
+E2005071978011634011406009729009729009472009496009498009312009312009312009400009426
+E2005071977013008012753012753012753012879012906012904012651012651012651011602011634
+E2005071976013212012953012953012953013076013105013103012846012846012846012971013008
+E2005071975013157013157013157013157013283013315013315013054013054013054013182013211
+E2005071974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005071973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005071972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005071971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005071970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005071969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005071968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005071967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005071966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005071965023411021474021579021577021577018597018597018042018135018135017840012812
+E2005071964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005071963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005071962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005071961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005071960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005071959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005071958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005071957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005071956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005071955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005071954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005071953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005071952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005071951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005071950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005071949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005071948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005071947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005071946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005071945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005071944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005071943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005071942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005071941                        009059009196009196009196009196009196009286009435
+S2005071970013671013671013671013671013804013870013870013870013870013870            
+S2005071969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005071968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005071967                        015333015404015405015405015405015405015552015628
+E2005081980009257009257009075009075009167009167                                    
+E2005081979009427009427009242009242009331009353009353009353009170009170009257009257
+E2005081978011634011634009729009729009472009496009498009498009312009312009400009426
+E2005081977013008013008012753012753012879012906012904012904012651012651011602011634
+E2005081976013212013212012953012953013076013105013103013103012846012846012971013008
+E2005081975013157013157013157013157013283013315013315013315013054013054013182013211
+E2005081974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005081973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005081972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005081971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005081970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005081969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005081968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005081967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005081966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005081965023411021474021579021577021577018597018597018308018135018135018135012812
+E2005081964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005081963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005081962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005081961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005081960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005081959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005081958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005081957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005081956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005081955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005081954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005081953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005081952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005081951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005081950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005081949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005081948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005081947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005081946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005081945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005081944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005081943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005081942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005081941                        009059009196009196009196009196009196009286009435
+S2005081970013671013671013671013671013804013870013870013870013870013870            
+S2005081969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005081968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005081967                        015333015404015405015405015405015405015552015628
+E2005091980009257009257009257009075009167009167                                    
+E2005091979009427009427009427009242009331009353009353009353009353009170009257009257
+E2005091978011634011634009891009729009472009496009498009498009498009312009400009426
+E2005091977013008013008013008012753012879012906012904012904012904012651011602011634
+E2005091976013212013212013212012953013076013105013103013103013103012846012971013008
+E2005091975013157013157013157013157013283013315013315013315013315013054013182013211
+E2005091974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005091973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005091972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005091971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005091970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005091969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005091968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005091967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005091966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005091965023411021474021579021577021577018597018597018308018403018135018135012812
+E2005091964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005091963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005091962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005091961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005091960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005091959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005091958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005091957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005091956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005091955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005091954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005091953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005091952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005091951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005091950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005091949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005091948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005091947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005091946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005091945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005091944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005091943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005091942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005091941                        009059009196009196009196009196009196009286009435
+S2005091970013671013671013671013671013804013870013870013870013870013870            
+S2005091969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005091968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005091967                        015333015404015405015405015405015405015552015628
+E2005101980009257009257009257009257009167009167                                    
+E2005101979009427009427009427009427009331009353009353009353009353009353009257009257
+E2005101978011634011634009891009891009472009496009498009498009498009498009400009426
+E2005101977013008013008013008013008012879012906012904012904012904012904011602011634
+E2005101976013212013212013212013212013076013105013103013103013103013103012971013008
+E2005101975013157013157013157013157013283013315013315013315013315013315013182013211
+E2005101974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005101973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005101972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005101971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005101970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005101969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005101968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005101967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005101966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005101965023411021474021579021577021577018597018597018308018403018403018135012812
+E2005101964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005101963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005101962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005101961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005101960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005101959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005101958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005101957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005101956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005101955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005101954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005101953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005101952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005101951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005101950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005101949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005101948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005101947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005101946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005101945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005101944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005101943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005101942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005101941                        009059009196009196009196009196009196009286009435
+S2005101970013671013671013671013671013804013870013870013870013870013870            
+S2005101969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005101968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005101967                        015333015404015405015405015405015405015552015628
+E2005111980009257009257009257009257009350009167                                    
+E2005111979009427009427009427009427009518009353009353009353009353009353009442009257
+E2005111978011634011634009891009891009662009496009498009498009498009498009588009426
+E2005111977013008013008013008013008013136012906012904012904012904012904011834011634
+E2005111976013212013212013212013212013337013105013103013103013103013103013230013008
+E2005111975013157013157013157013157013283013315013315013315013315013315013446013211
+E2005111974012848012848012848012848012971013002013002013002013002013002013126013159
+E2005111973012956012989012989012989012989013020013146013174013176013176013176012848
+E2005111972012588012617012619012619012619012654012777012805012805012805012805012833
+E2005111971012630012660012660012660012660012695012606012636012636012636012636012666
+E2005111970013700013734013736013736013736013800013936013969013969013969013969014001
+E2005111969013727013727013727013727013860013342013471013501013502013502013502013572
+E2005111968013128013128013128013128013257013416013414013414013414013414013544013728
+E2005111967012586012586012586012586012706012846012845012845012845012845012970013128
+E2005111966012813012813012194012194012202012328012327012327012327012327012446012585
+E2005111965023411021474021579021577021577018597018597018308018403018403018403012812
+E2005111964022455022671022779022777022777022929022929023152023261023261023261023411
+E2005111963021522021731021832021829021829021995021995022211022317022317022317022455
+E2005111962020675020875021027021024021024021079021079021281021432021433021433021522
+E2005111961019718019907020137020136020136020195020195020388020617020615020615020675
+E2005111960018850019017019236019236019236019269019269019452019678019679019679019718
+E2005111959019153019153019153019153019153018788018788018601018810018810018810018850
+E2005111958018312018312018312018312018312018453018672018671018671018671018671018934
+E2005111957019956019990019990019990019990020272020508020507020507020507020507018097
+E2005111956018679018679018859019383019383019429019429019429019619019848019847019956
+E2005111955017753017753017926018157018157018205018205018205018382018629018629018679
+E2005111954016886016886017051017269017269017312017312017312017481017716017717017753
+E2005111953017814017814016338016541016541016581016581016581016633016843016844016886
+E2005111952017066017066017066017066017338017378017378017378017546017769017766017814
+E2005111951016260016260016260016260016416016661016660016660016660016660016823017065
+E2005111950014706014706014706014706014848015053015053015053015053015053015199015426
+E2005111949013669013669013669013669013802014383014383014383014383014383014523014706
+E2005111948013141013141013141013141013109013323013323013323013323013323013451013669
+E2005111947012338012338012338012338012457012657012657012657012657012657012934013141
+E2005111946011728011728011728011728011841012025012025012025012025012025012142012338
+E2005111945011047011047011047011047011154011333011333011333011333011333011443011728
+E2005111944010509010509010509010509010611010770010770010770010770010770010875011047
+E2005111943009995009995009995009995010091010248010248010248010248010248010348010509
+E2005111942009435009435009435009435009600009745009745009745009745009745009840009995
+E2005111941                        009059009196009196009196009196009196009286009435
+S2005111970013671013671013671013671013804013870013870013870013870013870            
+S2005111969013570013570013570013570013437013500013500013500013500013500013607013670
+S2005111968015629015629015629015629014337014568014568014568013618013618013497013570
+S2005111967                        015333015404015405015405015405015405015552015628
+N2005062005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                                    
+N2005062004001291001287001284001281001282001278NO PAYNO PAYNO PAYNO PAYNO PAYNO PAY
+N2005062003001330001327001323001319001316001313001309001306001302001299001298001294
+N2005062002001382001379001375001372001364001360001356001353001349001346001337001334
+N2005062001001454001450001446001443001423001419001416001412001408001404001390001386
+N2005062000001546001541001537001533001517001513001497001493001489001485001462001458
+N2005061999001615001610001606001602001589001585001581001577001573001568001554001550
+N2005061998001704001699001695001690001667001662001657001653001648001644001624001619
+N2005061997001710001710001710001710001763001758001753001749001744001739001713001708
+N2005061996001791001791001791001791001775001775001748001748001748001748001736001736
+N2005061995001949001949001949001949001866001866001838001838001838001838001818001818
+N2005061994002036002036002036002036002025002025001995001995001995001995001979001979
+N2005061993002541002541002138002138002118002118002086002086002086002086002067002067
+N2005061992002644002644002644002644002644002644002592002592002592002592002592002592
+N2005061991002751002751002751002751002751002751002697002697002697002697002697002697
+N2005061990002862002862002862002862002862002862002806002806002806002806002806002806
+N2005061989002978002978002978002978002978002978002919002919002919002919002919002919
+N2005061988003098003098003098003098003098003098003037003037003037003037003037003037
+N2005061987003223003223003223003223003223003223003160003160003160003160003160003160
+N2005061986003730003730003730003730003730003730003657003657003657003657003288003288
+N2005061985003880003880003880003880003880003880003804003804003804003804003804003804
+N2005061984004170004170004170004170004045004045003980003980003980003980003958003958
+N2005061983005105005105004590004590004417004417004355004355004355004355004238004238
+N2005061982005566005566005566005566005566005566005457005457005457005457005207005207
+N2005061981006049006049006049006049005791005791005678005678005678005678005678005678
+N2005061980006798006798006798006798006866006866006732006732006732006732006170006170
+N2005072005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                              
+N2005072004001294001291001287001284001285001282001278NO PAYNO PAYNO PAYNO PAYNO PAY
+N2005072003001334001330001327001323001320001316001313001309001306001302001301001298
+N2005072002001386001382001379001375001367001364001360001356001353001349001341001337
+N2005072001001458001454001450001446001427001423001419001416001412001408001394001390
+N2005072000001550001546001541001537001522001517001513001497001493001489001466001462
+N2005071999001619001615001610001606001594001589001585001581001577001573001559001554
+N2005071998001708001704001699001695001671001667001662001657001653001648001628001624
+N2005071997001736001710001710001710001768001763001758001753001749001744001718001713
+N2005071996001818001791001791001791001775001775001775001748001748001748001736001736
+N2005071995001979001949001949001949001866001866001866001838001838001838001818001818
+N2005071994002067002036002036002036002025002025002025001995001995001995001979001979
+N2005071993002592002541002138002138002118002118002118002086002086002086002067002067
+N2005071992002697002644002644002644002644002644002644002592002592002592002592002592
+N2005071991002806002751002751002751002751002751002751002697002697002697002697002697
+N2005071990002919002862002862002862002862002862002862002806002806002806002806002806
+N2005071989003037002978002978002978002978002978002978002919002919002919002919002919
+N2005071988003160003098003098003098003098003098003098003037003037003037003037003037
+N2005071987003288003223003223003223003223003223003223003160003160003160003160003160
+N2005071986003804003730003730003730003730003730003730003657003657003657003288003288
+N2005071985003958003880003880003880003880003880003880003804003804003804003804003804
+N2005071984004238004170004170004170004045004045004045003980003980003980003958003958
+N2005071983005207005105004590004590004417004417004417004355004355004355004238004238
+N2005071982005678005566005566005566005566005566005566005457005457005457005207005207
+N2005071981006170006049006049006049005791005791005791005678005678005678005678005678
+N2005071980006934006798006798006798006866006866006866006732006732006732006170006170
+N2005082005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                        
+N2005082004001298001294001291001287001289001285001282001278NO PAYNO PAYNO PAYNO PAY
+N2005082003001337001334001330001327001323001320001316001313001309001306001305001301
+N2005082002001390001386001382001379001371001367001364001360001356001353001344001341
+N2005082001001462001458001454001450001431001427001423001419001416001412001397001394
+N2005082000001554001550001546001541001526001522001517001513001497001493001469001466
+N2005081999001624001619001615001610001598001594001589001585001581001577001563001559
+N2005081998001713001708001704001699001676001671001667001662001657001653001633001628
+N2005081997001736001736001710001710001773001768001763001758001753001749001723001718
+N2005081996001818001818001791001791001775001775001775001775001748001748001736001736
+N2005081995001979001979001949001949001866001866001866001866001838001838001818001818
+N2005081994002067002067002036002036002025002025002025002025001995001995001979001979
+N2005081993002592002592002138002138002118002118002118002118002086002086002067002067
+N2005081992002697002697002644002644002644002644002644002644002592002592002592002592
+N2005081991002806002806002751002751002751002751002751002751002697002697002697002697
+N2005081990002919002919002862002862002862002862002862002862002806002806002806002806
+N2005081989003037003037002978002978002978002978002978002978002919002919002919002919
+N2005081988003160003160003098003098003098003098003098003098003037003037003037003037
+N2005081987003288003288003223003223003223003223003223003223003160003160003160003160
+N2005081986003804003804003730003730003730003730003730003730003657003657003288003288
+N2005081985003958003958003880003880003880003880003880003880003804003804003804003804
+N2005081984004238004238004170004170004045004045004045004045003980003980003958003958
+N2005081983005207005207004590004590004417004417004417004417004355004355004238004238
+N2005081982005678005678005566005566005566005566005566005566005457005457005207005207
+N2005081981006170006170006049006049005791005791005791005791005678005678005678005678
+N2005081980006934006934006798006798006866006866006866006866006732006732006170006170
+N2005092005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                  
+N2005092004001301001298001294001291001293001289001285001282001278NO PAYNO PAYNO PAY
+N2005092003001341001337001334001330001327001323001320001316001313001309001309001305
+N2005092002001394001390001386001382001375001371001367001364001360001356001348001344
+N2005092001001466001462001458001454001435001431001427001423001419001416001401001397
+N2005092000001559001554001550001546001530001526001522001517001513001497001473001469
+N2005091999001628001624001619001615001603001598001594001589001585001581001568001563
+N2005091998001718001713001708001704001681001676001671001667001662001657001637001633
+N2005091997001736001736001736001710001778001773001768001763001758001753001727001723
+N2005091996001818001818001818001791001775001775001775001775001775001748001736001736
+N2005091995001979001979001979001949001866001866001866001866001866001838001818001818
+N2005091994002067002067002067002036002025002025002025002025002025001995001979001979
+N2005091993002592002592002170002138002118002118002118002118002118002086002067002067
+N2005091992002697002697002697002644002644002644002644002644002644002592002592002592
+N2005091991002806002806002806002751002751002751002751002751002751002697002697002697
+N2005091990002919002919002919002862002862002862002862002862002862002806002806002806
+N2005091989003037003037003037002978002978002978002978002978002978002919002919002919
+N2005091988003160003160003160003098003098003098003098003098003098003037003037003037
+N2005091987003288003288003288003223003223003223003223003223003223003160003160003160
+N2005091986003804003804003804003730003730003730003730003730003730003657003288003288
+N2005091985003958003958003958003880003880003880003880003880003880003804003804003804
+N2005091984004238004238004238004170004045004045004045004045004045003980003958003958
+N2005091983005207005207004666004590004417004417004417004417004417004355004238004238
+N2005091982005678005678005678005566005566005566005566005566005566005457005207005207
+N2005091981006170006170006170006049005791005791005791005791005791005678005678005678
+N2005091980006934006934006934006798006866006866006866006866006866006732006170006170
+N2005102005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY            
+N2005102004001305001301001298001294001296001293001289001285001282001278NO PAYNO PAY
+N2005102003001344001341001337001334001330001327001323001320001316001313001312001309
+N2005102002001397001394001390001386001379001375001371001367001364001360001352001348
+N2005102001001469001466001462001458001439001435001431001427001423001419001405001401
+N2005102000001563001559001554001550001535001530001526001522001517001513001477001473
+N2005101999001633001628001624001619001608001603001598001594001589001585001572001568
+N2005101998001723001718001713001708001686001681001676001671001667001662001642001637
+N2005101997001736001736001736001736001783001778001773001768001763001758001732001727
+N2005101996001818001818001818001818001775001775001775001775001775001775001736001736
+N2005101995001979001979001979001979001866001866001866001866001866001866001818001818
+N2005101994002067002067002067002067002025002025002025002025002025002025001979001979
+N2005101993002592002592002170002170002118002118002118002118002118002118002067002067
+N2005101992002697002697002697002697002644002644002644002644002644002644002592002592
+N2005101991002806002806002806002806002751002751002751002751002751002751002697002697
+N2005101990002919002919002919002919002862002862002862002862002862002862002806002806
+N2005101989003037003037003037003037002978002978002978002978002978002978002919002919
+N2005101988003160003160003160003160003098003098003098003098003098003098003037003037
+N2005101987003288003288003288003288003223003223003223003223003223003223003160003160
+N2005101986003804003804003804003804003730003730003730003730003730003730003288003288
+N2005101985003958003958003958003958003880003880003880003880003880003880003804003804
+N2005101984004238004238004238004238004045004045004045004045004045004045003958003958
+N2005101983005207005207004666004666004417004417004417004417004417004417004238004238
+N2005101982005678005678005678005678005566005566005566005566005566005566005207005207
+N2005101981006170006170006170006170005791005791005791005791005791005791005678005678
+N2005101980006934006934006934006934006866006866006866006866006866006866006170006170
+N2005112005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY      
+N2005112004001309001305001301001298001300001296001293001289001285001282001281NO PAY
+N2005112003001348001344001341001337001334001330001327001323001320001316001316001312
+N2005112002001401001397001394001390001383001379001375001371001367001364001355001352
+N2005112001001473001469001466001462001443001439001435001431001427001423001409001405
+N2005112000001568001563001559001554001539001535001530001526001522001517001494001477
+N2005111999001637001633001628001624001612001608001603001598001594001589001577001572
+N2005111998001727001723001718001713001690001686001681001676001671001667001647001642
+N2005111997001736001736001736001736001788001783001778001773001768001763001737001732
+N2005111996001818001818001818001818001804001775001775001775001775001775001764001736
+N2005111995001979001979001979001979001896001866001866001866001866001866001847001818
+N2005111994002067002067002067002067002058002025002025002025002025002025002009001979
+N2005111993002592002592002170002170002152002118002118002118002118002118002100002067
+N2005111992002697002697002697002697002697002644002644002644002644002644002644002592
+N2005111991002806002806002806002806002806002751002751002751002751002751002751002697
+N2005111990002919002919002919002919002919002862002862002862002862002862002862002806
+N2005111989003037003037003037003037003037002978002978002978002978002978002978002919
+N2005111988003160003160003160003160003160003098003098003098003098003098003098003037
+N2005111987003288003288003288003288003288003223003223003223003223003223003223003160
+N2005111986003804003804003804003804003804003730003730003730003730003730003353003288
+N2005111985003958003958003958003958003958003880003880003880003880003880003880003804
+N2005111984004238004238004238004238004118004045004045004045004045004045004037003958
+N2005111983005207005207004666004666004486004417004417004417004417004417004305004238
+N2005111982005678005678005678005678005678005566005566005566005566005566005311005207
+N2005111981006170006170006170006170005907005791005791005791005791005791005791005678
+N2005111980006934006934006934006934007004006866006866006866006866006866006294006170
+I2005062005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                                    
+I2005062004002587002579002571002564002573002565NO PAYNO PAYNO PAYNO PAYNO PAYNO PAY
+I2005062003002724002715002705002696002664002656002647002639002631002624002603002595
+I2005062002002845002834002823002813002804002793002782002772002761002751002743002734
+I2005062001003165003149003133003118003051003036003022003008002994002981002867002856
+I2005062000003445003428003411003394003400003380003311003294003277003261003196003180
+I2005061999003606003588003570003553003552003532003515003497003480003463003483003463
+I2005061998                                                003672003653003645003624
+I2005072005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                              
+I2005072004002595002587002579002571002581002573002565NO PAYNO PAYNO PAYNO PAYNO PAY
+I2005072003002734002724002715002705002672002664002656002647002639002631002611002603
+I2005072002002856002845002834002823002815002804002793002782002772002761002753002743
+I2005072001003180003165003149003133003065003051003036003022003008002994002878002867
+I2005072000003463003445003428003411003420003400003380003311003294003277003213003196
+I2005071999003624003606003588003570003572003552003532003515003497003480003503003483
+I2005071998                                                003690003672003665003645
+I2005082005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                        
+I2005082004002603002595002587002579002589002581002573002565NO PAYNO PAYNO PAYNO PAY
+I2005082003002743002734002724002715002681002672002664002656002647002639002619002611
+I2005082002002867002856002845002834002826002815002804002793002782002772002763002753
+I2005082001003196003180003165003149003079003065003051003036003022003008002889002878
+I2005082000003483003463003445003428003441003420003400003380003311003294003229003213
+I2005081999003645003624003606003588003593003572003552003532003515003497003523003503
+I2005081998                                                003709003690003686003665
+I2005092005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY                  
+I2005092004002611002603002595002587002599002589002581002573002565NO PAYNO PAYNO PAY
+I2005092003002753002743002734002724002691002681002672002664002656002647002629002619
+I2005092002002878002867002856002845002839002826002815002804002793002782002775002763
+I2005092001003213003196003180003165003096003079003065003051003036003022002902002889
+I2005092000003503003483003463003445003461003441003420003400003380003311003248003229
+I2005091999003665003645003624003606003613003593003572003552003532003515003544003523
+I2005091998                                                003727003709003707003686
+I2005102005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY            
+I2005102004002619002611002603002595002609002599002589002581002573002565NO PAYNO PAY
+I2005102003002763002753002743002734002702002691002681002672002664002656002639002629
+I2005102002002889002878002867002856002852002839002826002815002804002793002787002775
+I2005102001003229003213003196003180003113003096003079003065003051003036002916002902
+I2005102000003523003503003483003463003482003461003441003420003400003380003266003248
+I2005101999003686003665003645003624003634003613003593003572003552003532003564003544
+I2005101998                                                003749003727003728003707
+I2005112005NO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAYNO PAY      
+I2005112004002629002619002611002603002619002609002599002589002581002573002575NO PAY
+I2005112003002775002763002753002743002712002702002691002681002672002664002650002639
+I2005112002002902002889002878002867002865002852002839002826002815002804002799002787
+I2005112001003248003229003213003196003130003113003096003079003065003051002929002916
+I2005112000003544003523003503003483003502003482003461003441003420003400003343003266
+I2005111999003707003686003665003645003655003634003613003593003572003552003585003564
+I2005111998                                                003770003749003750003728
